tests/data-factory/LoginData.ts

```diff
@@ -0,0 +1,12 @@
+import Login from '../data-objects/Login';
+
+class LoginData {
+  validLoginData() {
+    const logindata = Login;
+    logindata.email = 'AutomationUser@mailinator.com';
+    logindata.password = 'automation@123';
+    return logindata;
+  }
+}
+
+export default new LoginData();
```

Comment:
Add this configuration to the eslint.json file
```
    "import/no-anonymous-default-export": ["error", {
      "allowArray": false,
      "allowArrowFunction": false,
      "allowAnonymousClass": false,
      "allowAnonymousFunction": false,
      "allowCallExpression": true, // The true value here is for backward compatibility
      "allowNew": false,
      "allowLiteral": false,
      "allowObject": false
    }],
    ```
    
    And refactor the codebase without anonymous default export

Reply:
Hello @lucky-gru 
Well, I couldn't find  eslint.json file , but I tired to put changes in .eslintrc.js it solves problem at code level but when I pushed code it gives me below error :

![image](https://github.com/etenlab/crowd.Bible/assets/5212585/00a5007f-40d1-4739-9052-cb44b71c8f1a)

![image](https://github.com/etenlab/crowd.Bible/assets/5212585/2f47e577-e9e5-4f59-82b3-47ac6e2c4d1f)

Am I missing anything ?